source/unix/openssl_platform_init.c

```diff
@@ -47,14 +80,15 @@ static void s_hmac_ctx_free(HMAC_CTX *ctx) {
 }
 
 /* libcrypto 1.0 shim for reset */
-static void s_hmac_ctx_reset(HMAC_CTX *ctx) {
+static int s_hmac_ctx_reset(HMAC_CTX *ctx) {
     AWS_PRECONDITION(ctx);
     AWS_PRECONDITION(
         g_aws_openssl_hmac_ctx_table->init_fn != s_hmac_ctx_init_noop &&
         g_aws_openssl_hmac_ctx_table->clean_up_fn != s_hmac_ctx_clean_up_noop &&
         "libcrypto 1.0 reset called on libcrypto 1.1 vtable");
     g_aws_openssl_hmac_ctx_table->clean_up_fn(ctx);
     g_aws_openssl_hmac_ctx_table->init_fn(ctx);
+    return true;
```

Comment:
I'd change this to a 1 with a note that it matches the semantics of HMAC_CTX_reset()

Reply:
Can do